web/public/locales/en.json

```diff
@@ -90,6 +90,23 @@
       "total-electricity-consumption": "Total electricity consumption by source"
     }
   },
+  "estimation-feedback": {
+    "title": {
+      "state-1": "Help us improve!",
```

Comment:
See my comment further down about naming the states, which I think should also be applied here :)